docs/developers/linux-rpm-demo.md

```diff
@@ -1,76 +1,11 @@
-### Run AWSOTelCollector Beta on Amazon Linux 2
+### Run ADOT Collector on Amazon Linux 2
+See [AWS Distro for OpenTelemetry documentation](https://aws-otel.github.io/docs/setup/build-collector-as-rpm) to run ADOT Collector on Amazon Linux 2 EC2 host.  
```

Comment:
This linked documentation appears to be missing instructions on how to verify the package integrity.

Reply:
That link is pointing to instructions of how to build the collector as RPM. does it make sense to check integrity for something that you just build yourself?
https://www.redhat.com/sysadmin/rpm-gpg-verify-packages

However you brought an interesting point. Do we have instructions on how to install from EC2 using the release buckets?

Reply:
We do but they point for docs in github https://aws-otel.github.io/docs/getting-started/collector -> https://github.com/aws-observability/aws-otel-collector/blob/main/docs/developers/linux-rpm-demo.md

Reply:
I have now added an alternative step in step 1 - to download rpm package from S3 and added the option to check for package integrity there. Please check this PR: https://github.com/aws-otel/aws-otel.github.io/pull/508. 
Thanks!